app/main.py

```diff
@@ -16,4 +16,9 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+#  Solution
```

Comment:
Please, remove the redundant comments. There should be no extra code in your PRs